instrumentation-genai/opentelemetry-instrumentation-openai-v2/examples/manual/main.py

```diff
@@ -24,6 +34,56 @@
     BatchSpanProcessor(OTLPSpanExporter())
 )
 
+# configure metrics
```

Comment:
As below are choosing boundaries, it could help in the example to explain why we are doing this, or at least what we are changing it, to.

For example, "# configure metrics for requests that are skewed slower/faster than the default bucket boundaries"
I'm not saying this is the right sentence, but we can take advantage of this to teach people why we are overriding the defaults. Otherwise, folks may just copy/paste this and now know why.